### PR TITLE
Added tooltip for the communications link

### PR DESCRIPTION
**Pull request by akshayasalvi**
### Fixed Issues

$ https://github.com/Expensify/App/issues/5542

### Tests

1. Tested with long emails
2. Tested for mobile layouts to ensure nothing breaks

### QA Steps

1. Navigate to a conversation with a user that has a long email
2. Open the user detail page
3. Hover the cursor over the email and you should see a tooltip

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

Screenshot 2021-11-01 at 2 57 31 PM

#### Mobile Web

NA

#### Desktop

Screenshot 2021-11-01 at 3 06 36 PM

#### iOS

NA

#### Android

NA

### CODE REVIEW

File: src/styles/styles.js [hunk 1]
```diff
@@ -2052,6 +2052,10 @@ const styles = {
         height: 20,
     },
 
+    communicationsLinkText: {
+        width: 320,
```
Comment (parasharrajat) [inline src/styles/styles.js:2056]: I am strictly against using the fixed width. I would prefer to keep the app flexible (using flexbox) as much as possible.

Any special purpose for using this?

File: src/components/CommunicationsLink.js [hunk 2]
```diff
@@ -52,7 +53,13 @@ const CommunicationsLink = props => (
                         >
                             {props.children}
                         </Pressable>
-                    ) : props.children}
+                    ) : (
+                        <Tooltip text={props.value}>
```
Comment (parasharrajat) [inline src/components/CommunicationsLink.js:57]: Ok. It works but it takes away flexibility that comes with `props.children`. This tooltip should be passed with children.

**Review by parasharrajat — CHANGES_REQUESTED** [review]

File: src/components/CommunicationsLink.js (hunk 2 above)
Comment (akshayasalvi) [reply src/components/CommunicationsLink.js:57]: Okay got it. I'll add a separate prop for Tooltip.

Comment (parasharrajat) [reply src/components/CommunicationsLink.js:57]: I think better would be to pass the tooltip with children

**Comment by akshayasalvi** [comment]
@parasharrajat PR updated

**Review by parasharrajat — APPROVED** [review]
LGTM.

All yours, @thienlnam .

**Review by thienlnam — APPROVED** [review]
Awesome, thank you both

**Comment by ogumen** [comment]
Accessibility issue found in this PR:
1. The tooltip bubble cannot be dismissed or hovered without moving mouse pointer - failure of WCAG SC 1.4.13, similar to https://github.com/Expensify/App/issues/5683
https://user-images.githubusercontent.com/88733897/142771959-cbcc6ac7-19db-4db8-a92f-88e87e29c1dc.mp4

**Pull request merged**